BenchLabs/interview-test
# Add a React and node apps

### Overview
Adds a README describing our interview test plus a React app and node.js app using Express.

### Changes
- Created a sample React app using Create React App that has some basic functionality for listing transactions, adding a transaction and a small report
- Created an equivalent node app with the same functionality.

### How to Test
1. Go into the `react-app` folder
2. Run `npm install`
3. Run `npm start`
4. Your browser should open up and you should be able to try out some features.
5. Go into the `node` folder
6. Run `npm install`
7. Run `npm start`
8. Run `curl --location 'http://localhost:3000/transactions'`
